app/models/invoice_item.rb

```diff
@@ -9,10 +11,20 @@ class InvoiceItem < ApplicationRecord
   validates :unit_price, presence: true
 
   def price_to_dollars
-    self.unit_price = self.unit_price / 100.00
+    if self.unit_price == self.unit_price.to_i
+      unit_price = self.unit_price.to_i
+    else
+      unit_price = self.unit_price
+    end
+
+    if unit_price.class != Float
+      self.unit_price = Money.new(unit_price, "USD")
+    else
+      self.unit_price = unit_price
+    end
   end
 
   def calculate_total
-    self.total = (self.quantity * self.unit_price).round(2) 
+    self.total = self.quantity * self.unit_price
```

Comment:
Ian suggested that round belongs in the serializer since that is the V in this project